### ISSUE TITLE
How'd you convince PayPal to let you work on open source?

### ISSUE THREAD

**Issue by kentcdodds** [kentcdodds/ama#440]
> Someone asked me this and prefers to remain anonymous.

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
